wgpu-hal/src/dx12/device.rs

```diff
@@ -1371,47 +1380,54 @@ impl crate::Device for super::Device {
                     };
                     size_of_val(&first_vertex) + size_of_val(&first_instance) + size_of_val(&other)
                 };
-                Some(super::CommandSignatures {
-                    draw: Self::create_command_signature(
+
+                let draw_mesh = if desc.flags.contains(crate::PipelineLayoutFlags::MESH_SHADER) {
```

Comment:
Instead of needing to pass this down as a flag, we should be able to check the wgpu-hal device for the currently enabled features

Reply:
Yes of course; I somehow missed (despite looking for it explicitly) that this function received `&self`.  Done.